templates/repo/sub_menu.tmpl

```diff
@@ -15,7 +15,10 @@
 					</div>
 				{{end}}
 				<div class="item">
-					<span class="ui">{{svg "octicon-database"}} <b>{{FileSize .Repository.Size}}</b></span>
+					<span class="ui{{if not (eq .Repository.LFSSize 0)}} tooltip{{end}}"{{if not (eq .Repository.LFSSize 0)}} data-content="{{.Repository.SizeDetailsString}}"{{end}}>
```

Comment:
```suggestion
					<span class="ui{{if not (eq .Repository.Size 0)}} tooltip{{end}}"{{if not (eq .Repository.Size 0)}} data-content="{{.Repository.SizeDetailsString}}"{{end}}>
```

Reply:
Same here not need for tool tip if no LFS.